### PR TITLE
Phase 2 route component set up

### PR DESCRIPTION
**Pull request by nickstaylor**
# PR Description

Please include a summary of the change:

This PR adds the function on the Configure component for storing a user quote in the redux store.  Also, the initial components for Historical were created, along with a route for the component.  Login component deleted and other commented lines deleted.  The Configure Component also has a temporary useEffect fetch call in order to gain further understanding to eventual data to be brought back.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
Creates object for user quote to be stored in redux store.  Next steps are to change the useEffect fetch calls to the BE once the BE is up and running
Note any questions or struggles as well.

## Type of change
Please unckeck options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Add Testing
- [ ] Refactor

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/containers/App/App.js [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
-    this.state = {
-      user: ''
-    }
-  }
 
-  render() {
     return (
       <div>
-        {this.state.user && <Header />}
+        {props.validatedUser && <Header />}
```
Comment (alex-latham) [inline src/containers/App/App.js:16]: Possibly related to the navigation bug preventing users from going back from the configure page?

**Comment by nickstaylor** [comment]
This creates #47 issue to be resolved

File: src/containers/Configure/Configure.js [hunk 2]
```diff
@@ -25,12 +29,31 @@ useEffect(() =>{
...
+    return updateError('Please fill all Inputs')
+  }
+  const quote = {
+              systemSize,
+              moduleType,
+              arrayType,
+              systemLosses,
+              tilt,
+              azimuth,
+              id: Date.now()
```
Comment (alex-latham) [inline src/containers/Configure/Configure.js:42]: Is the id used in the FE or is it being used similarly to a timestamp?

File: src/containers/Header/Header.css [hunk 3]
```diff
@@ -4,7 +4,9 @@
   justify-content: center;
   width: 100%;
   height: 4em;
-  background-image: linear-gradient(#6B9AC4, #EFF2F1);
+  background-color: whitesmoke;
+  opacity: .85;
+  /* background-image: linear-gradient(#6B9AC4, #EFF2F1); */
```
Comment (alex-latham) [inline src/containers/Header/Header.css:9]: Unused code

**Pull request merged**